frontend/src/components/Notification.tsx

```diff
@@ -0,0 +1,51 @@
+import { Snackbar, makeStyles } from '@material-ui/core'
+import * as React from 'react';
+import Alert from '@material-ui/lab/Alert';
+
+
+export type NotificationType =  {
+    isOpen: boolean;
+    type: 'error' | 'info' | 'success' | 'warning'; 
+    message: string;
+};
+
+
+
+const useStyles = makeStyles(theme => ({
+    root:{
+        top: theme.spacing(9) //Change this depending on size of header
+    }
+}));
+
+export default function Notification(props) {
+    const {notify, setNotify} = props
+    const classes = useStyles()
+
+    const handleClose = (e) => { 
+        setNotify({ 
+            ...notify,
+            isOpen: false
+        })
+    }
```

Comment:
This might be something the parent component would pass in, and probably as an optional prop

Reply:
It is optional now